### DISCUSSION TITLE
Students' Developer Pack

### DISCUSSION THREAD

**Discussion by Rashiahuja734** [community/community#61924] [GitHub Education]
### Hi everyone,

Greeting!
I am currently in 7th semester of B.Tech in National Institute of Technology Raipur. My current session will end in july 2024. My college doesn't provide email addresses to any of the student. I have applied for students' developer pack 3 times before, but it is still getting rejected. As a proof, I am submitting my 7th semester registration receipt which has a recent date.

**Comment by nirav-gajera** [comment]
try with  screenshot of college portal also as proof maybe it can work.

Found this here, https://education.github.com/discount_requests/application

**Reply by Rashiahuja734** [reply]
It still didn't work.

**Reply by nirav-gajera** [reply]
create support ticket https://support.github.com/request they'll help you

**Reply by Rashiahuja734** [reply]
It says "they cannot open ticket for this request"

**Reply by nirav-gajera** [reply]
try here : https://support.github.com/contact/education

**Reply by Rashiahuja734** [reply]
I was trying the same but when I am choosing "Application not approved (Student)" then it says the same thing.

**Reply by nirav-gajera** [reply]
![Screenshot (39)](https://github.com/community/community/assets/127472114/381d91e4-2bca-4f70-9327-5feb95142a96)
choose: Status of GitHub Education application (student)
 or go with  : Troubleshoot student developer pack  issue with our virtual agent

**Reply by Rashiahuja734** [reply]
Individual manual reviews and status updates are no longer provided...

**Reply by nirav-gajera** [reply]
tried this : Troubleshoot student developer pack issue with our virtual agent?

**Reply by Rashiahuja734** [reply]
yes... I am done with this now seriously. What kind of proof does this thing needs??

**Reply by nirav-gajera** [reply]
usually works with this : 
A copy of your current student ID card.
A screenshot of your school's website that shows your name and current enrollment status.
A fees receipt  
A letter from your school's registrar or academic advisor confirming your current enrollment status.

**Reply by Rashiahuja734** [reply]
I know... I am just frustrated. Btw thank you for helping me

**Reply by Rashiahuja734** [reply]
my Student Id card does not have any kind of date. Screenshot didn't work. Fee receipt didn't work.

**Reply by nirav-gajera** [reply]
NIT doesn't provide student mail id?

**Reply by Rashiahuja734** [reply]
NIT Raipur provides hostels to students on the basis of their grades, and you are asking about email id.

**Reply by Rashiahuja734** [reply]
I can apply for the email to my school, but it will take a long time.

**Reply by nirav-gajera** [reply]
then try continuously with id card, fees receipts, portal screenshot there is no other option.

**Reply by Rashiahuja734** [reply]
My application got rejected for 4 times today.

**Reply by nirav-gajera** [reply]
Watch the latest YouTube videos on this; you may find something there
